docs/source/pages/output_files.rst

```diff
@@ -7,4 +7,142 @@
 Output Files Guide
 ==================
 
-TBD
+.. note:: Please see :ref:`pages/cli_usage/index:common options` for the instructions on how to set the output directory of Macaron.
+
+-------------------
+Top level structure
+-------------------
+
+.. code-block::
+
+    output/
+        ├── .gradle/
+        ├── .m2/
+        ├── build_log/
+        ├── git_repos/
+        ├── reports/
+        ├── debug.log
+        ├── macaron.db
+        └── sbom_debug.json
+
+-------
+Reports
+-------
+
+The report files of Macaron (from using the :ref:`analyze action <analyze-action-cli>`) are generated into the ``reports`` directory.
+
+.. code-block::
+
+    output/
+    └── reports/
+        └── ...
+
+''''''''''''''''''
+Unique result path
+''''''''''''''''''
+
+Macaron creates a different directory under ``reports`` to store the report files for each target repository. This directory
+path is formed from the git host name (e.g ``github.com``), the owner and the name of that
+repository. The final path is created using the following template:
+
+.. code-block::
+
+    <path_to_output>/reports/<git_service_name>/<owner>/<repo_name>
+
+.. note:: The git host name has all occurrence of ``.`` in the URL replaced by ``_``.
+
+For example, when analyze the repository at `<https://github.com/micronaut-projects/micronaut-core>`_, Macaron will store
```

Comment:
```suggestion
For example, the reports for  `<https://github.com/micronaut-projects/micronaut-core>`_ repository will be stored under
```

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/281/commits/cacf960a62fb0430209e64eb22e21125a4a57760